docs/user/user/integrating-glean-for-product-managers.md

```diff
@@ -43,3 +43,12 @@ Occasionally, products need to collect data that doesn't fit neatly into one of
 Glean [has a process](https://wiki.mozilla.org/Glean/Adding_or_changing_Glean_metric_types) to request and introduce more metric types and we will work with you to design something appropriate.
 This design and implementation work is at least 4 weeks, though we are working on the foundation to accelerate that.
 Having a telemetry plan (see above) will help to identify this work early.
+
+## Integrating Glean into GLAM
+
+To use GLAM for analysis of your application's data [file a ticket in the GLAM repository](https://github.com/mozilla/glam/issues/new/choose) with the following information:
```

Comment:
Can we add a template to the GLAM repository?
That is: copy https://github.com/mozilla/glam/blob/main/.github/ISSUE_TEMPLATE/bug_report.md to something like `add_to_glam.md` and have the teplate ask for that info?